civictechprojects/static/css/partials/_ProjectFilterContainer.scss

```diff
@@ -138,3 +138,12 @@
     margin: 0;
   }
 }
+
+.favorite-filter {
+  font-weight: bold;
+  padding-left: 5px;
+
+  i {
+    color: $color-red
```

Comment:
Duplicate the style changes above, then add the 8px left/right margin requested - top and bottom are already handled with the parent padding so we don't need more of it here.

```css
i {
color: $color-favorite;
font-size: 24px;
margin: 0 8px;
```